.golangci.yml

```diff
@@ -0,0 +1,69 @@
+run:
+  build-tags: 
+    - aws
+    - packet
+    - aks
+    - e2e
+    - baremetal
+    - disruptivee2e
+    - poste2e
+
+  modules-download-mode: vendor
+
+  timeout: 5m
+
+issues:
+  exclude-use-default: false
+  new-from-rev: HEAD~
+  max-same-issues: 0
+  max-issues-per-linter: 0
+
+linters:
+  disable-all: true
+  enable:
```

Comment:
How do we update this list to include new linters?

Reply:
Just add new linters to list I think. If you are asking for new linters on new updates, we check it via `golangci-lint linters`. Also previously we were using `--enable-all` which is deprecated and will soon be removed. So I guess we had to move to config file

Reply:
I'd put a comment, that the list can be generated for updates using the following command:
```sh
golangci-lint linters | grep -v -E '(by your configuration linters|^$)' | cut -d: -f1 | cut -d ' ' -f1 | sort | sed 's/^/    - /g'
```